File: tools/mangl-packet-dump/gencmake.mjs
#! /usr/bin/env zx

import {generate} from '../../scripts/genproj/index.mjs';

let params = {
  buildConfig : '../../.build-config.yaml',
  sources : 'sources.yaml',
  targetCmakeSources : 'sources.cmake',
  targetCmakeConfig : '.build-config.cmake',
}

generate(params);

File: tools/mangl-animator/proj/gencmake.py
#! /usr/bin/env python3

from genproj import GenProj

proj = GenProj()

proj.buildConfig = "../../../build_config.yaml"
proj.sources = "Sources.yaml"
proj.targetCmakeSources = "Sources.cmake"
proj.targetCmakeConfig = ".build_config.cmake"

proj.generate()
proj.printResult()
